components/ArtPieces.js

```diff
@@ -1,23 +1,23 @@
 import styled from "styled-components";
 import ArtPiecePreview from "./ArtPiecePreview";
 
-const StyledList = styled.li`
+const StyledListItem = styled.li`
   list-style: none;
 `;
 
 export default function ArtPieces({ pieces }) {
   return (
-    <>
-      <h1> All art pieces:</h1>
+    <div>
```

Comment:
If you just need the `div` cause you want to render more than one child, I would recommend to use a fragment instead `<></>`.